Loader/ElFinderLoader.php

```diff
@@ -50,12 +50,18 @@ public function configure()
     /**
      * Starts ElFinder
      * @var $instance string
+     * @return void/array
      */
     public function load($instance)
     {
         $this->setInstance($instance);
-        $connector = new ElFinderConnector(new ElFinderBridge($this->configure()));
-        $connector->run();
+        $config = $this->configure();
+        $connector = new ElFinderConnector(new ElFinderBridge($config));
+        if ($config['corsSupport']) {
+            return $connector->run();
+        } else {
+            $connector->runAndExit();
+        }
```

Comment:
May be its better to swap function names to keep backward compatibility, i.e. run() for old and runAndExit()(may be renaming it?)?